packages/discovery-react-components/src/components/SearchResults/utils/getDocumentTitle.ts

```diff
@@ -0,0 +1,18 @@
+import DiscoveryV2 from 'ibm-watson/discovery/v2';
+import get from 'lodash/get';
+
+export const getDocumentTitle = (
+  resultTitleField: string,
+  result?: DiscoveryV2.QueryResult
+): string => {
```

Comment:
Switch order of params: the main object first, followed by the "qualifier".

Also, just because they were called "result" and "resultTitleField" in the earlier code doesn't mean we have to use those same names here. This is a standalone function and "result" makes no sense here. Use something like `document` and `titleField` instead.

Reply:
I think `titleField` should be optional in this function. So default (if no "overriding" field is specified) is: extracted title, file name, doc ID.

Reply:
for the first comment, the reason I had it in this order is because typescript doesn't like it when you have required parameter follow an optional parameter.

Reply:
Then either change it to be not optional (since that will be quite rare anyway, if at all) or change it's type to `QueryResult | null`.